cypress/e2e/racingcar.cy.js

```diff
@@ -0,0 +1,31 @@
+describe("lotto", () => {
+  beforeEach(() => {
+    cy.visit("http://127.0.0.1:5500/");
+  });
+
+  it("자동차 이름 작성 후, 게임이 시작되면 자동차명이 게임 화면에 같이 출력된다.", () => {
+    const carNames = ["red", "blue", "green"];
+    cy.get("#car-name-input").type(carNames.join(","));
+    cy.get("#car-name-btn").click();
+    cy.get("#try-count-input").type("3");
+    cy.get("#try-count-btn").click();
+
+    cy.get(".car-player").each((item, index) => {
+      expect(Cypress.$(item).text()).to.be.equal(carNames[index]);
+    });
+  });
+
+  it("5자 초과의 자동차 이름을 등록하면 경고창이 뜬다.", () => {
+    const alertStub = cy.stub();
+    cy.on("window:alert", alertStub);
+
+    cy.get("#car-name-input").type("red,blueeeee,green");
```

Comment:
```suggestion
    cy.get("#car-name-input").type("red, green");
```
이렇게 공백이 있을 경우에 6자로 체크되는 것 같아요!
이 부분도 한 번 확인해주세요~
![image](https://user-images.githubusercontent.com/18749057/181003033-ce46aeb9-9d24-4079-8b46-a7b5b93cd007.png)